pkg/inline/inline.go

```diff
@@ -31,15 +30,32 @@ import (
 // Inliner inlines data files by reading them from the local or a remote
 // filesystem.
 type Inliner struct {
-	// Local reader reads from the local filesystem.
-	Reader files.FileObjReader
+	// Reader reads from the local filesystem.
+	Readers   map[files.URLScheme]files.FileObjReader
```

Comment:
Make these private?  Doesn't seem like you need them to be public, and it makes me more concerned if they are.